### PR TITLE
Updates notebooks/official/custom/sdk-custom-image-classification-online.ipynb

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
Reduce boilerplate for sdk-custom-image-classification-online.ipynb notebook

- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/custom/sdk-custom-image-classification-online.ipynb [hunk 1]
```diff
@@ -142,342 +142,183 @@
       },
```
Comment (ivanmkc) [inline notebooks/official/custom/sdk-custom-image-classification-online.ipynb:142]: <pre class="ql-syntax" spellcheck="false">Line #9.    MODEL_DISPLAY_NAME = "cifar10-" + "unique"
</pre><p>Just use a string, no need to concatenate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (ivanmkc) [inline notebooks/official/custom/sdk-custom-image-classification-online.ipynb:142]: <pre class="ql-syntax" spellcheck="false">Line #1.    DEPLOYED_NAME = "cifar10_deployed-" + "unique"
</pre><p>Same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (sudarshan-SpringML) [reply notebooks/official/custom/sdk-custom-image-classification-online.ipynb:142]: done

Comment (sudarshan-SpringML) [reply notebooks/official/custom/sdk-custom-image-classification-online.ipynb:142]: done

**Review by ivanmkc — APPROVED** [review]
LGTM

**Pull request merged**